pkg/cmd/admin/policy/modify_roles.go

```diff
@@ -76,9 +76,13 @@ func NewCmdAddRoleToUser(name, fullName string, f *clientcmd.Factory, out io.Wri
 	saNames := []string{}
 
 	cmd := &cobra.Command{
-		Use:     name + " ROLE (USER | -z SERVICEACCOUNT) [USER ...]",
-		Short:   "Add a role to users or serviceaccounts for the current project",
-		Long:    `Add a role to users or serviceaccounts for the current project`,
+		Use:   name + " ROLE (USER | -z SERVICEACCOUNT) [USER ...]",
+		Short: "Add a role to users or serviceaccounts for the current project",
+		Long: fmt.Sprintf(`Add a role to users or serviceaccounts for the current project.
+
+To list existing cluster roles, you can run this command:
+
+$ oc get clusterroles`, fullName),
```

Comment:
> Admins running add-role-to-user may not be cluster admins, right?

`ClusterRoles` are exposed to everyone.  There are caveats.  For instance, assign a `system:***` is probably a mistake, unless its `system:image-puller` or `system:image-pusher`.  You also probably shouldn't assign roles with cluster scoped resources and you can't assign roles that have more power than you have.